addons/product/test/product_pricelist.yml

```diff
@@ -51,9 +51,9 @@
 -
   !python {model: product.product}: |
     from openerp.tools import float_compare
-    context.update({'quantity':1, 'date': False, 'partner': ref('base.res_partner_4'), 'pricelist': ref("supplier_pricelist")})
+    context.update({'quantity':1, 'date': False, 'partner': ref('base.res_partner_4')})
     product = self.browse(cr, uid, ref("product_product_6"), context=context)
-    assert float_compare(product.price, 792, precision_digits=2) == 0, "Wrong cost price: LCD Monitor."
+    assert float_compare(product.standard_price, 800, precision_digits=2) == 0, "Wrong cost price: LCD Monitor."
```

Comment:
must receive 790 (as the price_min_margin is not applied anymore) but we should still be able to select partner_4 (set it in the context) and receive its price